composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/App.kt

```diff
@@ -45,11 +58,54 @@ fun App(dao: DemoDao) {
     }
 
     nfcManager.startScan()
-    
+
     MRTBuddyTheme {
-        MainScreen(
-            cardState = McardState.value,
-            transactions = Mtransactions.value
-        )
+        var lang by remember { mutableStateOf(Language.English.isoFormat) }
+        LocalizedApp (
+            language= lang
+        ) {
+            Scaffold {
+                Box(
+                   Modifier.systemBarsPadding()
+                ) {
+                    Column {
+                        TopAppBar(
```

Comment:
This eats quite a bit of space from my small screen. Could we possibly move this to a Settings Screen? (We can land these changes in a feature branch and work from there I suppose?)